File: Nível 2/1046 - Tempo de Jogo/main.lua
valores = io.read()
x1, x2 = valores:match("(%S+)%s+(%S+)")
x1 = tonumber(x1)
x2 = tonumber(x2)

if x1 < x2 then 
    tempo = x2 - x1 
else
    tempo = (24 - x1) + x2 
end

print(string.format("O JOGO DUROU %d HORA(S)", tempo))

File: Nível 2/1046 - Tempo de Jogo/main.py
valores = input("A: ")

x1, x2 = valores.split()

x1 = int(x1)
x2 = int(x2)

if x1 < x2:
    tempo = x2 - x1
else:
    tempo = (24 - x1) + x2

print("O JOGO DUROU {} HORA(S)".format(tempo))